autoload/gista/client.vim

```diff
@@ -110,6 +110,10 @@ function! s:get_default_username(apiname) abort
     let username = get(g:gista#client#default_username, a:apiname, default)
   endif
   if empty(username)
+    let username = system('git config github.user')
```

Comment:
Please define `s:R = s:V.import('Process')` at the top of this file and use `s:R.system()` instead.

Additionally, please add a switch variable (say, `g:gista#client#use_git_config_github_user`) at [the bottom of this file](https://github.com/tyru/vim-gista/blob/git-config-github-user/autoload/gista/client.vim#L363) and use it to check if gista should call `git config github.user`.
It is required because

1. Calling `system` would slow the response
2. Some users may want to use vim-gista as an anonymous mode in default. This part is called when a new client is created, mean that it will be called by any `:Gista XXX` command so without switch option. That's why users require to call `:Gista logout` everytime after vim started if they want to use vim-gista as an anonymous mode in default.

And please add documentation about the variable around

https://github.com/lambdalisue/vim-gista/blob/master/doc/vim-gista.txt#L124
https://github.com/lambdalisue/vim-gista/blob/master/doc/vim-gista.txt#L256
https://github.com/lambdalisue/vim-gista/blob/master/doc/vim-gista.txt#L471